src/fidesops/main.py

```diff
@@ -47,6 +51,21 @@ def start_webserver() -> None:
     if config.database.ENABLED:
         logger.info("Starting scheduled request intake...")
         initiate_scheduled_request_intake()
+    import os
+
+    if os.getenv("FIDESOPS__ROOT_USER__ANALYTICS_ID"):
+        logger.info(
+            f"Analytics ID override active for internal use. Will send events with client id = {os.getenv('FIDESOPS__ROOT_USER__ANALYTICS_ID')}"
```

Comment:
Theoretically anyone could set this ENV variable, so it won't necessarily be "for internal use". Regardless, if you offload the `client_id` population to `config.py` as suggested above, you won't need this log line at all.